core/spoofax.compiler/src/main/resources/mb/spoofax/compiler/platform/intellij_project/Component.java.mustache

```diff
@@ -18,7 +21,8 @@ import mb.spoofax.intellij.log.IntellijLoggerComponent;
         IntellijLoggerComponent.class,
         {{adapterProjectCompilerInput.resourcesComponent.qualifiedId}}.class,
         ResourceServiceComponent.class,
-        IntellijPlatformComponent.class
+        IntellijPlatformComponent.class{{#adapterProjectCompilerInput.tegoRuntime}},
+        TegoComponent.class{{/adapterProjectCompilerInput.tegoRuntime}}
```

Comment:
Why is the `TegoComponent` only a dependency in the IntelliJ language component? And does this not conflict with the `TegoRuntime` exposed in the module of the language? Did you test this codepath? I think that Dagger will mark this as a compile error.